lib/hubstats/github_api.rb

```diff
@@ -87,6 +87,35 @@ def self.update_pulls
       end
     end
 
+    # Public - Gets information on teams, e.g. name
+    #
+    # Returns - nothing
+    def self.update_teams
+      grab_size = 250
+      begin
+        client = Hubstats::GithubAPI.client
+        incomplete = client.organization_teams("sportngin")
+        team_list = Hubstats.config.github_config["team_list"]
+
+        incomplete.each do |team|
+          if team_list.include? team[:name]
+            users = client.team_members(team[:id])
+            team[:action] = "added"
+            users.each do |user|
+              team[:current_user] = user
+              puts "Adding users to a team and/or making a team"
```

Comment:
The logging here could be more informative and accurate.  The team upsert should happen in the team block, and this message should be about a single user being added to a team.